pkg/cmd/cli/restore/create.go

```diff
@@ -38,7 +38,7 @@ func NewCreateCommand(f client.Factory, use string) *cobra.Command {
 	o := NewCreateOptions()
 
 	c := &cobra.Command{
-		Use:   use + " BACKUP",
+		Use:   use + " [NAME] --from-backup <backup-name>",
```

Comment:
Not sure this is entirely clear, open to input on how best to document that restore name is optional, --from-backup is required

Reply:
For the `Use` I think this is the form I would expect.  If you wanted to elaborate on it, I think the `Short`, `Long` or `Examples` would be the best place to add that additional context

Reply:
I would put `--from-backup` first, e.g. `--from-backup BACKUP_NAME [RESTORE_NAME]`. Putting something in `[braces]` typically denotes that it's optional. And +1 to Justin's suggestion of using short/long/example to show what happens both with and without the restore name.